packaging/cloudify-agents.spec

```diff
@@ -28,6 +28,7 @@ Vendor:         Cloudify Platform Ltd.
 Packager:       Cloudify Platform Ltd.
 
 Requires(pre):  shadow-utils
+Requires:       python3 >= 3.6
```

Comment:
I think we're targeting minimum 3.10?

Reply:
In fact, this isn't a build-requires, it's install-time, but we bring our own python so I think this line is just wrong?

Reply:
Needed for copy-packages at the moment. We probably want to nix this at some point, but let's leave it for now.